app/controllers/static_pages_controller.rb

```diff
@@ -2,22 +2,58 @@ class StaticPagesController < ApplicationController
   # GET / (root)
   # GET /static_pages/home
   def home
-    @app_name = Settings.app.name
-    @app_version = Settings.app.version
-    @registration_enabled = Settings.features.registration
+    return unless logged_in?
+
+    @pagy, @courses = pagy(
+      current_user.courses
+                  .by_status(params[:status])
+                  .ordered_by_start_date,
+      items: Settings.ui.items_per_page
+    )
+  end
+
+  private
+  def trainee_dashboard
+    courses = current_user.courses
+    courses = courses.where(status: params[:status]) if params[:status].present?
+    @pagy, @courses = pagy(courses.ordered_by_start_date,
+                           items: Settings.ui.items_per_page)
```

Comment:
```suggestion
    @pagy, @courses = pagy(
      current_user.courses.by_status(params[:status]).ordered_by_start_date,
      items: Settings.ui.items_per_page
    )
```
Nối scope luôn cho gọn code e nha